src/Apps/Settings/Routes/EditProfile/Components/SettingsEditProfileImage/SettingsEditProfileImage.tsx

```diff
@@ -0,0 +1,81 @@
+import { Flex, Text } from "@artsy/palette"
+import { CollectorProfileHeaderAvatarFragmentContainer } from "Apps/CollectorProfile/Components/CollectorProfileHeader/Components/CollectorProfileHeaderAvatar"
+import { EditProfileFormModel } from "Apps/Settings/Routes/EditProfile/Components/SettingsEditProfileFields"
+import { LocalImagePreview } from "Apps/Settings/Routes/EditProfile/Components/SettingsEditProfileImage/Components/LocalImagePreview"
+import { useFormikContext } from "formik"
+import { ChangeEvent, useState } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+import { SettingsEditProfileImage_me$data } from "__generated__/SettingsEditProfileImage_me.graphql"
+
+interface SettingsEditProfileImageProps {
+  me: SettingsEditProfileImage_me$data
+}
+
+const SettingsEditProfileImage: React.FC<SettingsEditProfileImageProps> = ({
+  me,
+}) => {
+  const [localImage, setlocalImage] = useState<string>("")
+  const { setFieldValue } = useFormikContext<EditProfileFormModel>()
+
+  const handleChange = async (event: ChangeEvent<HTMLInputElement>) => {
+    if (!event.target.files) return
+
+    const newImage = event.target.files[0]
+    const imagePreview = URL.createObjectURL(newImage)
+
+    setlocalImage(imagePreview)
+    setFieldValue("photo", newImage)
```

Comment:
**suggestion:** I think it's fine if this goes to the state but it will be gone if the user refreshes. Have you considered maybe using the helpers I added here https://github.com/artsy/force/blob/main/src/Utils/localImagesHelpers.ts
you will probably need a new key but it should be straightforward and you can save the image to the store on success